clang/lib/Driver/ToolChains/Darwin.cpp

```diff
@@ -1509,6 +1511,11 @@ void DarwinClang::AddLinkRuntimeLibArgs(const ArgList &Args,
         AddLinkSanitizerLibArgs(Args, CmdArgs, "asan");
       }
```

Comment:
Driver needs tests

Reply:
Thanks for catching this! Added a few basic ones that test that the flag is supported, and that some known sanitizer incompatibilities are blocked